UI/styles.css

```diff
@@ -0,0 +1,387 @@
+}
+
+.ad_text {
+    font-size: 60px;
+    font-weight: bold;
+    color: white;
+    font-family: Epilogue;
+    margin-top: 65px;
+    margin-right: 140px;
+}
+
+/* Filter */
+.filter {
+    display: flex;
+    width: 100%;
+    height: 250px;
+    border-top: 2px solid #9fb1fe;
+    border-bottom: 2px solid #9fb1fe;
+}
+
+.date_filter {
+    display: block;
+    width: 25%;
+    height: 250px;
+}
+
+.hashtags_filter {
+    display: block;
+    width: 50%;
+    height: 250px;
+}
+
+.vendor_filter {
+    display: block;
+    width: 25%;
+    height: 250px;
+}
+
+.filter_title {
+    font-family: Epilogue;
+    font-size: 45px;
+    color: #9fb1fe;
+    text-align: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    font-weight: bold;
+}
+
+.calendar_block {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 85px;
+    height: 65px;
+    font-family: Epilogue;
+    font-size: 30px;
+    color: #9fb1fe;
+    font-weight: bold;
+}
+
+.calendar {
+    color: #e1e6ff;
+    font-family: Epilogue;
+    font-size: 15px;
+    border: 1px solid #9fb1fe;
+    margin-bottom: 20px;
+    text-align: center;
+    font-weight: normal;
+}
+
+.select_vendor_button {
+    padding: 25px 80px;
+    color: #e1e6ff;
+    font-family: Epilogue;
+    font-size: 20px;
+    border: 1px solid #9fb1fe;
+    margin-left: 70px;
+    margin-top: 10px;
+}
+
+.hashtags_blocks {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 30px;
+    color: #9fb1fe;
+    font-family: Epilogue;
+    font-size: 30px;
+    line-height: 1.5;
+}
+
+/* Main */
+.row {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.left_add {
+    display: block;
+    width: 50%;
+    border-right: 2px solid #9fb1fe;
+}
+
+.right_add {
+    display: block;
+    width: 50%;
+}
+
+.ad_title {
+    font-family: Epilogue;
+    font-size: 45px;
+    color: #9fb1fe;
+    text-align: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    font-weight: bold;
+}
+
+.ad_info {
+    display: flex;
+    justify-content: space-between;
+}
+
+.pic_container {
+    direction: block;
+    width: 200px;
+    height: 200px;
+    border: 1px solid #9fb1fe;
+    margin-left: 30px;
+}
+
+.title_buttons {
+    display: flex;
+    justify-content: center;
+}
+
+.delete {
+    margin-right: 40px;
+}
+
+.description_container {
+    display: block;
+    height: 80px;
+    width: 300px;
+    text-align: center;
+    color: #9fb1fe;
+    font-family: Helvetica;
+    font-size: 23px;  
+}
+
+.more {
+    color: #957af5;
+    font-size: 24px;
+}
+
+.discount_container {
+    text-align: center;
+    font-family: Epilogue;
+    font-weight: bold;
+    font-size: 36px;
+    color: #7c95ff;
+    margin-top: 5px;
+    margin-right: 20px;
+}
+
+.discount {
+    font-size: 100px;
+    color: #c67cff;
+    margin: 10px;
+}
+
+.hashtags {
+    margin-top: 20px;
+    margin-left: 20px;
+    margin-bottom: 10px;
+    font-family: Epilogue;
```

Comment:
In case you have several fonts on the same page you can set default one to 'html' or 'body' tag and others to particular tags.